frontend/src/Utilities/String/translate.ts

```diff
@@ -27,7 +27,9 @@ export default function translate(
   key: string,
   tokens?: Record<string, string | number | boolean>
 ) {
-  const translation = translations[key] || key;
+  let translation = translations[key] || key;
+
+  translation = translation.replace(/\{(appName)\}/gi, 'Sonarr');
```

Comment:
I don't think we should replace tokens outside of the normal means, I'm not sure we need to pass in `appName` in every situation this is used, but should use tokens to replace it.

Reply:
Your're going to make me work for this one I see :D

Reply:
Hey, I gave you a shortcut to just add it to tokens if it's not there 😄